README.org

```diff
@@ -146,7 +146,7 @@
    | Erlang                | [[https://github.com/erlang-ls/erlang_ls][erlang_ls]]                                     | Yes            | [[https://github.com/erlang-ls/erlang_ls][erlang_ls]]                                                                                               |                              |
    | F#                    | [[https://github.com/fsharp/FsAutoComplete][fsautocomplete]]                                | Yes            | Automatic by [[https://github.com/emacs-lsp/lsp-mode/blob/master/lsp-fsharp.el][lsp-fsharp]]                                                                                 | No                           |
    | Fortran               | [[https://github.com/hansec/fortran-language-server][fortran-language-server]]                       | Yes            | pip install fortran-language-server                                                                     | Yes                          |
-   | Go                    | [[https://golang.org/x/tools/cmd/gopls][gopls]]                                         | Yes            | go get golang.org/x/tools/gopls@latest                                                                  | Yes                          |
```

Comment:
It would be better put `https://github.com/golang/tools/tree/master/gopls` in the `Language Server` column, and put `https://github.com/golang/tools/blob/master/gopls/doc/user.md` in the `Installation command` column. Your opinion?

Reply:
Yes, that seems like a better project homepage to link to.